csharp/ql/test/query-tests/Security Features/CWE-020/options

```diff
@@ -1 +1,3 @@
-semmle-extractor-options: /r:${testdir}/../../../resources/assemblies/System.Web.dll /r:${testdir}/../../../resources/assemblies/System.Web.ApplicationServices.dll /r:${testdir}/../../../resources/assemblies/System.Data.dll /r:System.Text.RegularExpressions.dll /r:System.Collections.Specialized.dll /r:System.Data.Common.dll /r:System.Security.Cryptography.X509Certificates.dll /r:System.Runtime.InteropServices.dll
+semmle-extractor-options: /nostdlib /noconfig
+semmle-extractor-options: --load-sources-from-project:${testdir}/../../../resources/stubs/_frameworks/Microsoft.NETCore.App/Microsoft.NETCore.App.csproj
+semmle-extractor-options: ${testdir}/../../../resources/stubs/System.Web.cs
```

Comment:
Does this file not belong to a properly generated stub from a nuget package? (We can take that follow-up).

Reply:
The content of `System.Web.cs`?
I agree that the most ideal situation would be to only rely on generated stuff.

Reply:
Yes; `System.Web.cs` appears to be handwritten; it would be nice if we could generate that using `make_stubs_nuget.py`.

Reply:
Yes, agreed, but I think that file contains stuff from multiple packages that has just piled up over time.